hnn_core/network.py

```diff
@@ -1665,15 +1672,33 @@ def _add_cell_type_bias(network: Network, amplitude: Union[float, dict],
                              f'{list(network.cell_types.keys())}. '
                              f'Got {cell_type}')
 
-        if 'tonic' not in network.external_biases:
-            network.external_biases['tonic'] = dict()
+        if bias_name not in network.external_biases:
+            network.external_biases[bias_name] = dict()
 
-        if cell_type in network.external_biases['tonic']:
-            raise ValueError(f'Tonic bias already defined for {cell_type}')
+        if cell_type in network.external_biases[bias_name]:
+            raise ValueError(f'Bias named {bias_name} already defined '
+                             f'for {cell_type}')
```

Comment:
This will prevent re-defining a tonic that has already been added. I think it made sense when there could only be one tonic but the with the behavior now it may make sense to just overwrite the existing tonic if it exists. Thoughts @jasmainak @ntolley?

Reply:
this behavior is present in the external drives too? probably best not to offer new features to the users unintentionally ... I tend to update the dictionary elements directly to update the drives/biases but it does mean we need to document it properly. Either way, we should discuss in a separate issue what is the recommended way to update drives/biases ...